src/plugins/hetero/executable_network.cpp

```diff
@@ -59,30 +63,25 @@ template <typename T>
 using NodeMap = std::unordered_map<ngraph::Node*, T>;
 
 HeteroExecutableNetwork::HeteroExecutableNetwork(const InferenceEngine::CNNNetwork& network,
-                                                 const Configs& user_config,
-                                                 Engine* plugin)
+                                                 const ov::AnyMap& user_config,
+                                                 const std::shared_ptr<ov::hetero::Plugin>& plugin)
     : InferenceEngine::ExecutableNetworkThreadSafeDefault(nullptr,
                                                           std::make_shared<InferenceEngine::ImmediateExecutor>()),
-      _heteroPlugin{plugin},
-      _name{network.getName()},
-      _hetero_config{},
-      _device_config{} {
+      _plugin{plugin},
+      _name{network.getName()} {
     auto function = network.getFunction();
     auto clonedFunction = ngraph::clone_function(*function);
 
-    // hetero_config, device_config and user_config are unchanged global and local configs set by user
-    // we need to create _hetero_config and _device_config based on them, which will
-    // contain only hetero (_hetero_config) and only device (_device_config) properties
-    auto parsed_config = _heteroPlugin->MergeConfigs(user_config);
-    _hetero_config = parsed_config.hetero_config;
-    _device_config = parsed_config.device_config;
+    // Split `user_config` on `device_config` (with properties for underlying devices)
+    // and `_cfg` (hetero config with properties for hetero plugin only)
+    auto device_config = user_config;
```

Comment:
I think you don't need to have this local variable anymore.

Reply:
Because `Configuration` erases "hetero" properties from `user_config` in order to leave only "device" properties, I can't to pass "const" object and have to create copy